pennylane/_device.py

```diff
@@ -337,6 +345,23 @@ def expval(self, expectation, wires, par):
         """
         raise NotImplementedError
 
+    @abc.abstractmethod
+    def var(self, expectation, wires, par):
+        """Return the variance of an expectation.
+
+        For plugin developers: this function should return the variance of the given
+        expectation on the device.
+
+        Args:
+            expectation (str): name of the expectation
```

Comment:
Note: if we rename the `expval` module to `observables`, it will also make the docstrings cleaner, and allow us to fix up our terminology
```python
        Args:
            observable (str): name of the observable
```

Reply:
How do we reconcile with adding non-expval measurements? i.e., single shots?

Reply:
That is, we have a module of observables, but the user accesses them via one of `qml.expval`, `qml.var`, or `qml.measure`